kaldb/src/test/java/com/slack/kaldb/logstore/LuceneIndexStoreImplTest.java

```diff
@@ -223,8 +223,9 @@ public void indexLongUnbreakableField() {
       MessageUtil.addFieldToMessage(msg, "hugefield", hugeField);
       logStore.logStore.addMessage(msg);
       assertThat(getCount(MESSAGES_RECEIVED_COUNTER, logStore.metricsRegistry)).isEqualTo(1);
-      assertThat(getCount(MESSAGES_FAILED_COUNTER, logStore.metricsRegistry)).isEqualTo(0);
-      // Counters not set since no commit.
+      // UTF8 encoding is longer than the max length 32766
```

Comment:
since we now default to STRING instead of TEXT ( which is in my opinion the right default for texty fields ) the entire high field is encoded as a single token and lucene has a limit. So the document actually errors out